### ISSUE TITLE
Report runs good in Designer, but no data in IIS

### ISSUE THREAD

**Issue by Quttia** [ariacom/Seal-Report#24]
Report result runs good in Seal Report Designer, and log is normal

But when runs in IIS, there is no result data, and log lack of 'Executing main query...', did I left something?

Thanks for the answers!

**In Seal Report Designer, normal**
![image](https://user-images.githubusercontent.com/26808114/60170279-b8f37980-983a-11e9-8e02-f51d3085669c.png)

**In IIS, no data**
![image](https://user-images.githubusercontent.com/26808114/60170596-5484ea00-983b-11e9-8d4e-5b50b96182c7.png)

**In Seal Report Designer, normal**
![20190626180013](https://user-images.githubusercontent.com/26808114/60171028-51d6c480-983c-11e9-9000-7db84205205d.png)

**In IIS, lack of 'Executing main query...'**
![image](https://user-images.githubusercontent.com/26808114/60170772-c3624300-983b-11e9-96af-1b5de39bc98c.png)

**Comment by ariacom** [comment]
did you use the last 5.0 ? check that there is no JavaScript error or missing in your browser (F12 to view errors)

**Comment by Quttia** [comment]
nope
![image](https://user-images.githubusercontent.com/26808114/60172346-1093e400-983f-11e9-914b-cab79df861c7.png)

**Comment by Quttia** [comment]
and in IIS, simple is normal too.

![image](https://user-images.githubusercontent.com/26808114/60172465-4c2eae00-983f-11e9-9911-417410a5aa16.png)

**Comment by Quttia** [comment]
only my personal report file runs wrong


[New report111.zip](https://github.com/ariacom/Seal-Report/files/3329532/New.report111.zip)

**Comment by ariacom** [comment]
is it related to your Data Source definition, using IIS the user has not the same rights...

**Comment by Quttia** [comment]
I used ODBC to connect Oracle, and checking connection is ok

![image](https://user-images.githubusercontent.com/26808114/60173745-08897380-9842-11e9-9901-8cace0eae342.png)

**Comment by ariacom** [comment]
you have to understand what is the difference between reports built your Data Source and Reports built with Northwind, did you run the Report Designer in Administrator mode to edit you report ?

**Comment by Quttia** [comment]
The issue is solved, the problem is that when I deploy SealWebServer in IIS, I changed RepositoryPath node of Web.config to another directory, but when I added a new data source on SealServerManger, the new.srex source file was default save in "C:\ProgramData\Seal Report Repository", so data source file new.srex was not detected when web run. Now I copy the new.srex to the designated path, it worked.
![image](https://user-images.githubusercontent.com/26808114/60246705-d9cad600-98f1-11e9-96b5-56dfb7619ff7.png)

Anyway, thanks for your advices so much, and your report project is such a masterpiece!
